hooks/persistence-azure-monitor/hook/hook.test.js

```diff
@@ -0,0 +1,83 @@
+// SPDX-FileCopyrightText: 2022 iteratec GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+process.env = Object.assign(process.env, {"MONITOR_WORKSPACE_ID": '123123', 
+                                          "MONITOR_SHARED_KEY": "ffffffffffffffffffffffffffffff",
+                                          "MONITOR_LOGTYPE_PREFIX": "SCB"});
```

Comment:
Seems simpler to me :)

```suggestion
process.env["MONITOR_WORKSPACE_ID"] = "123123";
process.env["MONITOR_SHARED_KEY"] = "ffffffffffffffffffffffffffffff";
process.env["MONITOR_LOGTYPE_PREFIX"] = "SCB";
```